drivers/wifi/eswifi/eswifi_core.c

```diff
@@ -350,20 +364,17 @@ static void eswifi_iface_init(struct net_if *iface)
 {
 	struct eswifi_dev *eswifi = &eswifi0;
 	u8_t mac[6];
-	int err;
 
 	LOG_DBG("");
 
 	eswifi_lock(eswifi);
 
-	err = eswifi_reset(eswifi);
-	if (err) {
+	if (eswifi_reset(eswifi)) {
```

Comment:
This and below looks unclear. Zephyr uses a convention for using negative numbers for errors. So, please use `if (eswifi_reset(eswifi) < 0)`. If eswifi_reset() doesn't follow the convention, well, it's worth fixing. And this whole function (eswifi_iface_init()) really should return error code itself.

Reply:
ok thanks